changelog.d/269.feature.md

```diff
@@ -0,0 +1,4 @@
+The `parse_json` function now accepts an optional `lossy` parameter (which defaults to `true`).
+This new parameter allows to control whether the UTF-8 decoding should be lossy or not, replacing
+invalid utf-8 sequences with the Unicode replacement character (U+FFFD) if set to `true` or raising an error
+if set to `false` and an invalid utf-8 sequence is found.
```

Comment:
```suggestion
The `parse_json` function now accepts an optional `lossy` parameter (which defaults to `true`).

This new parameter allows to control whether the UTF-8 decoding should be lossy or not, replacing
invalid utf-8 sequences with the Unicode replacement character (U+FFFD) if set to `true` or raising an error
if set to `false` and an invalid utf-8 sequence is found.
```

It seems like you were going for two paragraphs here so I split this up so it renders as such.

Reply:
Yes, missed that! Thank you!